packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -21,7 +22,7 @@ class MethodChannelCamera extends CameraPlatform {
   final Map<int, MethodChannel> _channels = {};
 
   /// The controller we need to broadcast the different events coming
-  /// from handleMethodCall.
+  /// from handleMethodCall. Specific to camera events.
```

Comment:
nit: I think this can be updated to one sentence:

```suggestion
  /// from handleMethodCall, specific to camera events.
```